tema14Ianuarie/src/com/sci/bv/tema14Ianuarie/Main.java

```diff
@@ -0,0 +1,133 @@
+                            "-------------------------------------------------------------\n");
+            x = scanner.nextInt();
+            if (x == 1)  // If the user chooses first option
+            {
+                System.out.println(
+                        "-------------------------------------------------------------\n" +
+                                "Product type (1 - animal; 2 - vegetable):\n" +
+                                "-------------------------------------------------------------\n" +
+                                "\n");
+                x = scanner.nextInt();
+                while (x == 1) {
+                    Animlist = myList.addAnimalProduct(); //Add animal product and characteristics
+                    System.out.println(
+                            "-------------------------------------------------------------\n" +
+                                    "1.Add another animal product\n" +
+                                    "0.Go back to main menu\n" +
+                                    "-------------------------------------------------------------\n" +
+                                    "\n");
+                    x = scanner.nextInt();
+                }
+                while (x == 2) {
+                    Veglist = myList1.addVegetableProduct();//Add animal product and characteristics
+                    System.out.println("-------------------------------------------------------------\n" +
+                            "2.Add another vegetable product\n" +
+                            "0.Go back to main menu\n" +
+                            "-------------------------------------------------------------\n" +
+                            "\n");
+                    x = scanner.nextInt();
+                }
+
+            }
+            while (x == 2) {
+                System.out.println("-------------------------------------------------------------\n" +
+                        "ID   Quantity   Product's name      Optimal Temperature    Price    Validity    Weight\n");
+                myList.displayProduct(); //Display animal products and characteristics
+                System.out.println("-------------------------------------------------------------\n" +
+                        "ID   Quantity   Product's name      Vitamins List    Price    Validity    Weight\n");
+                myList1.displayProduct();//Display vegetable products and characteristics
+                System.out.println("Please write the ID product:");
+                int ID = scanner.nextInt();
+                System.out.println("Please write the quantity:");
+                int Qty = scanner.nextInt();
+                for (AnimalProduct a : Animlist) {
+                    if (ID == a.getProductID()) { // Verify if the given product's ID is on stock
+                        if (Qty <= a.getProductQty() && a.getProductQty() != 0) {//Verify if is in stock the given quantity
```

Comment:
the second check does not help here... if Qty is <= than the productQty, just sell it. 
A better second check would be to see if Qty is positive so you're not selling -5 milk.